test/unit/specs/components/transactions/TmLiAnyTransaction.spec.js

```diff
@@ -8,7 +8,7 @@ describe(`TmLiAnyTransaction`, () => {
   const propsData = {
     transaction: transactions[0],
     validators: state.candidates,
-    address: `tb1da6xsetjg9jxgun9wdesexv05j`,
+    address: `A`,
```

Comment:
lol I just meant switching the prefix (eg: `cosmos1da6xsetjg9jxgun9wdesexv05j` instead of `tb1da6xsetjg9jxgun9wdesexv05j`)

Reply:
but nbd